### PR TITLE
Implemented user/conversation/message deletion

### PR DESCRIPTION
**Pull request by alxchen**
on clientside for command line, excluding Stores

### CODE REVIEW

File: src/codeu/chat/client/ClientConversation.java [hunk 1]
```diff
@@ -103,6 +104,24 @@ public void startConversation(String title, Uuid owner) {
...
+  // For c-remove command.
+  public void removeConversation(String title) {
+
+    ConversationSummary cs = summariesSortedByTitle.first(title);
+    if (cs == null) { // conversation does not exist for user
+      System.out.format("Error: conversation \"%s\" does not exist\n", title);
+      return;
+    }
+    summariesByUuid.remove(cs.id);
+    //summariesSortedByTitle.remove(cs.title); // Need to update with BTree store
```
Comment (ataylor4) [inline src/codeu/chat/client/ClientConversation.java]: should be summariesSortedByTitle = summariesSortedByTitle.remove(cs.title)

File: src/codeu/chat/client/ClientConversation.java [hunk 2]
```diff
@@ -103,6 +104,24 @@ public void startConversation(String title, Uuid owner) {
...
+
+    if (!summariesByUuid.containsKey(cs.id)) {
+      LOG.info("Conversation removed: Title= \"%s\"\n", title);
+      System.out.format("Conversation removed: Title= \"%s\"\n", title);
+
```
Comment (ataylor4) [inline src/codeu/chat/client/ClientConversation.java]: np: extra newline

File: src/codeu/chat/client/commandline/Chat.java [hunk 3]
```diff
@@ -270,6 +301,10 @@ private void addUser(String name) {
     clientContext.user.addUser(name);
   }
 
+  // Delete an existing user.
+  private void removeUser(String name) { clientContext.user.removeUser(name);}
```
Comment (ataylor4) [inline src/codeu/chat/client/commandline/Chat.java]: should be whitespace

**Review by ataylor4 — CHANGES_REQUESTED** [review]
Add unit tests and minor nitpicks, otherwise LGTM

**Review by rkinini — APPROVED** [review]
lgtm

**Pull request merged**